common/src/commonMain/kotlin/com/darkrockstudios/apps/hammer/common/components/encyclopedia/Encyclopedia.kt

```diff
@@ -19,17 +22,28 @@ interface Encyclopedia : Router {
 		data class CreateEntryDestination(val component: CreateEntry) : Destination()
 	}
 
-	sealed class Config : Parcelable {
-		@Parcelize
+	@Serializable
+	sealed class Config {
+		@Serializable
 		data class BrowseEntriesConfig(val projectDef: ProjectDef) : Config()
 
-		@Parcelize
+		@Serializable
 		data class ViewEntryConfig(val entryDef: EntryDef) : Config()
 
-		@Parcelize
+		@Serializable
 		data class CreateEntryConfig(val projectDef: ProjectDef) : Config()
 	}
 
+	object ConfigSerializer : KSerializer<Config> by polymorphicSerializer(
```

Comment:
This serializer looks redundant. Perhaps you could just use `Config.serializer()`.

Reply:
Ya I was wondering that, I think I misread the comment about polymorphic serialization